lib/logthrdestdrv.c

```diff
@@ -76,24 +76,21 @@ log_threaded_dest_worker_rewind_messages(LogThreadedDestWorker *self, gint batch
   self->batch_size -= batch_size;
 }
 
-static const gchar *
+static gchar *
```

Comment:
Why did you need to remove const? `g_strdup_printf` indeed returns `gchar *`, but I think that is converted automatically without warnings to `const gchar *`.